src/utils/utils.ts

```diff
@@ -3,6 +3,22 @@ export const timeStamp = (): string => {
   return now.toLocaleString();
 };
 
+const idChars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+export const createMessageId = (): string => {
```

Comment:
Seems like this should have a comment saying why you're not just using a `nanoid` or some other standard random identifier.

Reply:
Ah, I hadn't thought of that - I just added in the nanoid package and am using that now.